Jellyfin.Server/Program.cs

```diff
@@ -35,14 +37,30 @@ public static class Program
 
         public static async Task Main(string[] args)
         {
-            StartupOptions options = new StartupOptions(args);
-            Version version = Assembly.GetEntryAssembly().GetName().Version;
-
-            if (options.ContainsOption("-v") || options.ContainsOption("--version"))
+            // For backwards compatibility.
+            // Modify any input arguments now which start with single-hyphen to POSIX standard
+            // double-hyphen to allow parsing by CommandLineParser package.
+            const string pattern = @"^(-[^-\s]{2})"; // Match -xx, not -x, not --xx, not xx
+            const string substitution = @"-$1"; // Prepend with additional single-hyphen
+            var regex = new Regex(pattern);
+
+            for (var i = 0; i < args.Length; i++)
             {
-                Console.WriteLine(version.ToString());
+                args[i] = regex.Replace(args[i], substitution);
             }
 
+            // For CommandLine package, change default behaviour to output errors to stdout (instead of stderr)
+            var parser = new Parser(config => config.HelpWriter = Console.Out);
```

Comment:
hmm, why print errors to stdout?..

Reply:
I thought stdout was sensible for some reason.  Remove and pipe to stderr (which is default)?

Reply:
My thoughts are that errors should go to `stderr` if there's no strong reason for doing otherwise. End user can always redirect `stderr` to `stdout` if they desire so...

Reply:
Removed this line and lets use default behaviour.